src/NHibernate/AdoNet/ConnectionManager.cs

```diff
@@ -41,6 +42,10 @@ public interface Callback
 		private readonly ISessionImplementor session;
 		private readonly ConnectionReleaseMode connectionReleaseMode;
 		private readonly IInterceptor interceptor;
+		private readonly List<ISessionImplementor> _sessionsSharingManager = new List<ISessionImplementor>();
```

Comment:
Can this be renamed to something more obvious? I don't understand who is managing whom from the name.